runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -30,14 +36,21 @@ object GrpcClientSettings {
   }
 
   /** Scala API */
-  def apply(config: Config): GrpcClientSettings =
-    GrpcClientSettings(config getString "host", config getInt "port")
+  def apply(config: Config): GrpcClientSettings = {
+    var settings = GrpcClientSettings(config getString "host", config getInt "port")
       .copy(
         overrideAuthority = getOptionalString(config, "override-authority"),
         deadline = getPotentiallyInfiniteDuration(config, "deadline"),
-        trustedCaCertificate = getOptionalString(config, "trusted-ca-certificate"),
         userAgent = getOptionalString(config, "user-agent"))
 
+    // TODO: Decide whether to present a deprecation warning for this option.
```

Comment:
this hasn't even been included in a release yet as far as I know, so deprecation not needed